### DISCUSSION TITLE
Concurrent Request to Single Prometheus Server

### DISCUSSION THREAD

**Discussion by shreyank06** [prometheus/prometheus#13902] [Q&A]
I have 2 software components A and B which send their time series metrics to the same prometheus database.

I want to have 2 clients A and B which will talk to the same prometheus server for requesting software components A and B's data concurrently. 

Basically i want to have 2 clients A and B which want to talk to same prometheus server concurrently for fetching metrics.

Is it possible?

**Comment by machine424** [comment]
Prometheus in multi-client, I suggest you give it a try ;)
